spec/feature/hearings/daily_docket/build_hearsched_spec.rb

```diff
@@ -150,4 +150,93 @@
       include_context "Scheduled in Error hearing is removed from daily docket"
     end
   end
+
+  shared_context "fnod_badge display" do
```

Comment:
```suggestion
  shared_examples "fnod_badge display" do
```
`shared_context` is for blocks that set data up for tests; `shared_examples` is for blocks that contain expectations.